generatedTypes/services/LogService.d.ts

```diff
@@ -1,8 +1,8 @@
 declare function warn(message: string): void;
-declare function deprecationWarn({ component, oldProp, newProp }?: ({
-    component?: string;
-    oldProp?: string;
-    newProp?: string;
+declare function deprecationWarn({ component, oldProp, newProp }: ({
+    component: string;
+    oldProp: string;
+    newProp: string;
```

Comment:
'newProp' parameter should stay optional as we sometimes deprecate a prop with no replacement. Which means we also need to edit the massage accordingly...

Reply:
Added an option